themes/SuiteP/css/Dawn/style.scss

```diff
@@ -81,5 +81,4 @@
 @import '../suitep-base/studio';
 @import '../suitep-base/tinemce.scss';
 @import '../suitep-base/jstree';
-@import '../suitep-base/search';
```

Comment:
Removing the base search? we'd need this for BC but it shouldn't need removal anyway.